cargo-dist/src/config/version.rs

```diff
@@ -0,0 +1,67 @@
+//! For determining which configuration version we're using.
+
+use axoasset::SourceFile;
+use camino::Utf8PathBuf;
+use serde::Deserialize;
+
+use crate::{DistError, DistResult};
+
+const MAX_VERSION: i64 = 1;
```

Comment:
thought- instead of a const i64... do we want to make this a semver version and/or a enum? the validity check with "greater than" feels non specific and picking a different type for this value feels like the right call. personally i am leaning towards just using semver